src/08_comprehensions.py

```diff
@@ -10,14 +10,14 @@
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [x for x in range(1, 6)]
 
-print (y)
+print(y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
+y = [x**2 for x in range(10)]
```

Comment:
Looking for cubes here, not squares. Just try to make sure that if you're given a target output, you're checking to try to hit it exactly